CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Change Log
 ==========
 
+## Version 1.7.0
+- Better crash type reporting on various UBSan issues.
+- Initial support for Honggfuzz.
+- Additional fixes in preparation for migration to Python 3.
+- Improved HWASan support for Android.
```

Comment:
Can subsitute with something like
- Added libFuzzer fuzzing support for Android (using HWASan).

Reply:
Also,
- Migrated off deprecated App Engine Memcache to Cloud MemoryStore for Redis.

Reply:
something on invalid directory auto-fxing